mplc/utils.py

```diff
@@ -12,8 +12,19 @@
 from loguru import logger
 from ruamel.yaml import YAML
 
+import warnings
+from pathlib import Path
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import pylab
+
 from . import constants
 
+sns.set_style("white")
+pylab.rcParams.update({'font.size': 18})
+
```

Comment:
Could you add a comment/title to these 2 lines so that they don't sit here isolated? 😃